models/enterprise_sonic/route_maps/sonic_route_maps.yml

```
@@ -0,0 +1,383 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'community',
+      'license': 'Apache 2.0'
+    }
+NETWORK_OS: sonic
+RESOURCE: route_maps
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: sonic_route_maps
+  version_added: "2.1.0"
+  author: "Kerry Meyer (@kerry-meyer)"
+  short_description: route map configuration handling for SONiC
+  description:
+    - This module provides configuration management for route map parameters on devices running SONiC.
+  options:
+    config:
+      description:
+        - Specifies a list of route map configuration dictionaries
+      type: list
+      elements: dict
+      options:
+        name:
+          description:
+            - Name of a route map
+          type: str
+          required: true
+        action:
+          description:
+            - action type for the route map (permit or deny)
+          type: str
+          required: true
+          choices:
+            - permit
+            - deny
+        sequence_num:
+          description:
+            - unique number in the range 1-66535 to specify priority of the map
+          type: int
+          required: true
+        match:
+          description: Criteria for matching the route map to a route
+          type: dict
+          options:
+            as-path:
+              description:
+                - Name of a configured BGP AS path list to be checked for
+                - a match with the target route
+              type: str
+            community:
+              description:
+                - Name of a configured BGP "community" to be checked for
+                - a match with the target route
+              type: str
+            evpn:
+              description:
+                - BGP Ethernet Virtual Private Network to be checked for
+                - a match with the target route
+              type: dict
+              required_one_of: [['default-route', 'route-type', 'vni']]
+              options:
+                default_route:
+                  description:
+                    - Default EVPN type-5 route
+                  type: bool
+                route-type:
```

```suggestion
                route_type:
```
Hyphenated variable names aren't valid in playbooks, use underscore instead. Please correct all hyphenated variables in this file.

---

I have made this change in all files (this one and all example files). Thanks for reminding me of the reason for requiring the use of '_' in Ansible argument names.